slide_analysis/descriptors/descriptor_class.py

```diff
@@ -1,15 +1,4 @@
-from slide_analysis.utils.tile_class import Tile
-
-
 class Descriptor:
```

Comment:
As I understand Descriptor should be abstract class with abstract methods _calc,compare(otherDescriptor) and predifened method get_value()

Reply:
smth like this 
```python
from slide_analysis.utils.tile_class import Tile
from abc import ABC, abstractmethod


class Descriptor(ABC):
    def __init__(self, tile):
        self.tile = tile
        self._calc()
        self.value = None

    @abstractmethod
    def _calc(self):
        pass

    def get_value(self):
        if self.value is None:
            self._calc()

        return self.value

    @abstractmethod
    def compare(self, other_descriptor):
        pass
```